plugin/src/main/java/com/gtnewhorizons/retrofuturagradle/modutils/DependencyDeobfuscationTransform.java

```diff
@@ -1,28 +1,13 @@
 package com.gtnewhorizons.retrofuturagradle.modutils;
 
-import java.io.BufferedInputStream;
-import java.io.BufferedOutputStream;
-import java.io.File;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
+import java.io.*;
 import java.nio.file.Path;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.jar.Attributes;
-import java.util.jar.JarEntry;
-import java.util.jar.JarInputStream;
-import java.util.jar.JarOutputStream;
-import java.util.jar.Manifest;
+import java.util.*;
+import java.util.jar.*;
```

Comment:
we try to avoid star imports but maybe that flag isn't on yet, anyway, it shouldn't need to change (configure IDE to not compress them)

Reply:
Yeah, change your IDE wildcard import threshold to 999 and reorganize imports in this file to fix.